app/src/main/java/gholzrib/arctouchchallenge/core/utils/LoadingDialog.java

```diff
@@ -0,0 +1,40 @@
+package gholzrib.arctouchchallenge.core.utils;
+
+/**
+ * Created by Gunther Ribak on 08/01/2015.
+ * For more information contact me
+ * through guntherhr@gmail.com
+ */
+
+import android.app.ProgressDialog;
+import android.content.Context;
+
+public class LoadingDialog extends ProgressDialog {
+
+    private static final String MESSAGE_LOADING = "Loading";
+
+    private Context context;
+    private boolean isCancelable;
+
+    public LoadingDialog(Context context, int theme, boolean isCancelable) {
+        super(context, theme);
+        this.context = context;
+        this.isCancelable = isCancelable;
+        setupDialog();
+    }
+
+    public LoadingDialog(Context context, boolean isCancelable) {
+        super(context);
+        this.context = context;
+        this.isCancelable = isCancelable;
```

Comment:
The _isCancelable_ exists to offer the developer this option. I tried to develop the classes with a little more than needed, in case the project is continued and new features are added.